lib/commands/general.js

```diff
@@ -159,16 +159,28 @@ commands.getStrings = async function (language) {
     log.info(`No language specified, returning strings for: ${language}`);
   }
 
+  // Clients require the resulting mapping to have both keys
+  // and values of type string
+  const preprocessStringsMap = (mapping) => {
+    const result = {};
+    for (const [key, value] of _.toPairs(mapping)) {
```

Comment:
This could be done with `_.reduce`, and not produce an extra array. Something like
```js
const preprocessStringsMap = function preprocessStringsMap (mapping) {
  return _.reduce(mapping, function (result, key, value) {
    result[key] = _.isString(value) ? value : JSON.stringify(value);
    return result;
  }, {});
};
```

Reply:
np, changed